appengine/headless-chrome/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "headless-chrome-sample",
+  "engines": {
+    "node": "8.x"
+  },
+  "version": "1.0.0",
+  "description": "An example of taking screenshot with Puppeteer (Headless Chrome) in Node.js on Google App Engine.",
+  "scripts": {
+    "start": "node app.js",
+    "system-test": "repo-tools test app",
```

Comment:
We only have one test but we're using it both as system-test and as unit-test :wink: It should be run as one or the other.

Reply:
I am not sure I understand your comment. Can you be more specific about what I should do?

We have 2 different kind of tests runnign here. both of them are run when running `nom run test`

Reply:
But there's only one test file with one test. Where's the second test?

Reply:
Please compare to the other samples in this repository, I tried to mimic the architecture  as much as possible.
The test run with `system-test` is defined in `package.json` in `cloud-repo-tools`

Reply:
See for example https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/appengine/datastore